pkgs/top-level/octave-packages.nix

```
@@ -0,0 +1,58 @@
+# This file contains the GNU Octave add-on packages set.
+# Each attribute is an Octave library.
+# Expressions for the Octave libraries are supposed to be in `pkgs/development/octave-modules/<name>/default.nix`.
+
+{ pkgs
+, lib
+, stdenv
+, fetchurl
+, newScope
+, octave
+}:
+
+with lib;
+
+makeScope newScope (self:
+  let
+    inherit (octave) blas lapack gfortran python texinfo gnuplot;
+
+    callPackage = self.callPackage;
+
+    buildOctaveLibrary = callPackage ../development/interpreters/octave/mk-octave-derivation.nix {
+      inherit pkgs lib stdenv;
+      namePrefix = "${octave.pname}-${octave.version}";
+      inherit octave;
+    };
+
+  in {
+
+    inherit callPackage buildOctaveLibrary;
+
+    control = callPackage ../development/octave-modules/control { };
+
+    io = callPackage ../development/octave-modules/io {
+      unzip = pkgs.unzip;
+    };
+
+    level-set = callPackage ../development/octave-modules/level-set { };
+
+    linear-algebra = callPackage ../development/octave-modules/linear-algebra { };
+
+    ltfat = callPackage ../development/octave-modules/ltfat {
+      fftw = pkgs.fftw;
+      fftwSinglePrec = pkgs.fftwSinglePrec;
+      fftwFloat = pkgs.fftwFloat;
+      fftwLongDouble = pkgs.fftwLongDouble;
+      portaudio = pkgs.portaudio;
```

Many packages here are also used in octave's build. So we need to add a passthru to octave and use here:

```suggestion
      inherit (octave) fftw fftwSinglePrec portaudio;
      inherit (pkgs) fftwFloat;
```

As explained for other deps previously.